tests/OrcidIdTest.php

```diff
@@ -22,4 +22,9 @@ public function testSupportsOrcidsEndingInLowercaseX()
     {
         $this->assertEquals(['0000-0002-1694-233X'], OrcidId::extract('0000-0002-1694-233x'));
     }
+
+    public function testExtractsOrcidsWithTrailingUnicodePunctuation()
+    {
+        $this->assertEquals(['0000-0002-0088-0058'], OrcidId::extract('orcid.org/0000-0002-0088-0058…'));
```

Comment:
I guess `OrchidId` doesn't need this although its doesn't have the unicode support in its regex? https://github.com/altmetric/php-identifiers/blob/master/src/OrcidId.php#L9